confluence-xml/src/main/java/org/xwiki/contrib/confluence/filter/internal/input/ConfluenceInputFilterStream.java

```diff
@@ -253,6 +265,42 @@ private void readInternal(Object filter, ConfluenceFilter proxyFilter) throws Fi
         closeConfluencePackage();
     }
 
+    private void generateBlogEvents(Long blogSpaceId, Object filter, ConfluenceFilter proxyFilter)
+        throws FilterException
+    {
+        FilterEventParameters spaceParameters = new FilterEventParameters();
+
+        ConfluenceProperties blogSpaceProperties;
+
+        try {
+            blogSpaceProperties = this.confluencePackage.getSpaceProperties(blogSpaceId);
+        } catch (ConfigurationException e) {
+            throw new FilterException("Failed to get space properties", e);
+        }
+
+        String blogSpaceKey = toEntityName(this.properties.getBlogSpaceName());
+
+        FilterEventParameters blogSpaceParameters = new FilterEventParameters();
```

Comment:
If you don't have any parameter to associated to the space, use FilterEventParameters.EMPTY.